test/API/H5_api_dataset_test.h

```diff
@@ -149,7 +149,7 @@ int H5_api_dataset_test(void);
 #define DATASET_IO_POINT_DSET_NAME_NOCHUNK "dataset_io_point_selection_dset_nochunk"
 #define DATASET_IO_POINT_DSET_NAME_CHUNK   "dataset_io_point_selection_dset_chunk"
 
-#ifndef NO_LARGE_TESTS
+/*#ifndef NO_LARGE_TESTS
```

Comment:
All the NO_LARGE_TESTS code we can go ahead and remove since those tests aren't great and one was buggy to begin with. Hopefully t_bigio and others can eventually fulfill the role of these tests.

Reply:
Okay, great. I can remove those as well.